hbase-client/src/main/java/org/apache/hadoop/hbase/client/BufferedMutatorImpl.java

```diff
@@ -142,7 +142,9 @@ public class BufferedMutatorImpl implements BufferedMutator {
     RpcControllerFactory rpcFactory, BufferedMutatorParams params) {
     this(conn, params,
       // puts need to track errors globally due to how the APIs currently work.
-      new AsyncProcess(conn, conn.getConfiguration(), rpcCallerFactory, rpcFactory));
+      // todo rmattingly support buffered mutator request attributes
+      new AsyncProcess(conn, conn.getConfiguration(), rpcCallerFactory, rpcFactory,
+        Collections.emptyMap()));
```

Comment:
I think it was an oversight not to implement request attribute support for the buffered mutator in https://github.com/apache/hbase/pull/5326. I can introduce that here now and then port to master

Reply:
Still need to do this, will do before I mark as ready for review

Reply:
can you remove this todo?